src/dds/rs-dds-auto-calibration.h

```diff
@@ -0,0 +1,40 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <src/auto-calibrated-device.h>
+
+
+namespace librealsense
+{
+
+class dds_auto_calibrated : public auto_calibrated_interface
```

Comment:
Nothing here needs "DDS"... right?
I.e., this is technically just a `auto_calibrated_proxy`, no?

Reply:
And if so, then it does not belong in dds/

Reply:
done